app/components/CurrencyPicker.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import {
+  Menu,
+  MenuOptions,
+  MenuOption,
+  MenuTrigger,
+} from 'react-native-popup-menu';
+import currencies from '../constants/currencies';
+import styles from '../styles/FormStyles';
+
+const CurrencyPicker = ({ onValueChange, selectedValue = currencies[0] }) => {
+  const getLabel = ({ name, sign }) => `${name}(${sign})`;
+
+  const getCurrenciesListItems = () => currencies.map(value => (
```

Comment:
create separate named function instead anonymous (value => ...) for better performance